src/views/SecondAssessment.vue

```diff
@@ -0,0 +1,101 @@
+<script setup lang="ts">
+import { ref, onMounted } from 'vue'
+import { useRolesStore } from '@/stores/roles'
+import { ElSlider, ElButton } from 'element-plus'
+import type { Role } from '@/domain/Role'
+import { quizAnswer } from '@/stores/roles/state'
+import { ErrorMessage, SuccessMessage } from '@/services/messages'
+import type { QuizAnswers } from '@/domain/QuizAnswer'
+
+const roleId = ref('1')
+const rolesStore = useRolesStore()
+const role = ref<Role>({ name: '', knowledges: [], levels: [] })
+
+onMounted(async () => {
+  role.value = await rolesStore.getRoleById(roleId.value)
+  rolesStore.changeQuizAnswers(role.value.knowledges ?? [], quizAnswer.value)
+})
+
+const getResult = async (quizAnswer: QuizAnswers) => {
+  if (!quizAnswer) {
+    ErrorMessage('No data provided')
+  }
+  SuccessMessage('Fetching your results...')
+  try {
+    await rolesStore.getQuizResult(quizAnswer)
+    SuccessMessage('Success getting your result!')
```

Comment:
You don't need this message, when the result is recovered, the page will load the result, so the user will know the operation is completed